features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1263,6 +1263,18 @@
         <EnableRoleValidation>{{application_mgt.enable_role_validation}}</EnableRoleValidation>
     </ApplicationMgt>
 
+    {% if outbound_provisioning_management.reset_provisioning_entities_on_config_update is defined %}
+        <!--
+        Disabling this configuration will allow to keep the provisioned entities as it is and  will update the outbound
+        provisioning config rather than deleting the existing entries and add them as the new entries whenever updating
+        the provisioning connector configurations.
+        The default value of the configuration is true.
+        -->
+         <OutboundProvisioning>
+            <ResetProvisioningEntitiesOnConfigUpdate>{{outbound_provisioning_management.reset_provisioning_entities_on_config_update}}</ResetProvisioningEntitiesOnConfigUpdate>
+         </OutboundProvisioning>
```

Comment:
Shall we fix the formatting here?

Reply:
shall we align the config with the above configs

Reply:
It already aligned with the above configs right?